web/client/components/manager/rulesmanager/RuleAttributeSelect.jsx

```diff
@@ -114,9 +114,12 @@ const RuleAttributeSelect = React.createClass({
                     }
                     this.props.loadOptions();
                 }}
+                searchable={true}
+                matchPos="any"
+                matchProp="any"
                 clearable={this.props.clearable}
                 onInputChange={(input) => {
-                    if (!this.props.staticValues) {
+                    if (!this.props.staticValues && (!options && !this.props.paginated)) {
```

Comment:
I see, My fault, simply we should not display loading, because this when the operation is not async, (most of the times) this make the filtering do not work. Going to fix it too